### PR TITLE
Fix sql format documentation

### LINKED ISSUE #1487: Sql Format documentation needs update when reading from a database
# Overview

In `frictionless` [Sql Format docs](https://framework.frictionlessdata.io/docs/formats/sql.html) the code snippet for reading a data package from a SQL database is 

```python
from frictionless import Package
package = Package.from_sql('postgresql://database')
```

which generates the error `AttributeError: type object 'Package' has no attribute 'from_sql'`. The correct way (at least in version 5.10.5) is simply:

```python
from frictionless import Package
package = Package('postgresql://database')
```

### PR DESCRIPTION
**Pull request by shashigharti**
- fixes #1487
---

Please make sure that all the checks pass. Please add here any additional information regarding this pull request. It's highly recommended that you link this PR to an issue (please create one if it doesn't exist for this PR)

### CODE REVIEW

File: docs/formats/sql.md [hunk 1]
```diff
@@ -15,7 +15,8 @@ You can read SQL database:
 from pprint import pprint
 from frictionless import Package
 
-package = Package.from_sql('postgresql://database')
+control = formats.SqlControl(table="language", basepath="data")
```
Comment (roll) [inline docs/formats/sql.md]: @shashigharti Why do we need control here? :smiley:

Comment (shashigharti) [reply docs/formats/sql.md]: Thanks! I have removed that.

**Comment by shashigharti** [comment]
@roll Thank you!. I have made the changes.

**Comment by roll** [comment]
Thanks!

**Pull request merged**